db/schema.rb

```diff
@@ -399,8 +399,9 @@
   create_table "viewers", force: :cascade do |t|
     t.string   "viewer_type"
     t.integer  "exhibit_id"
-    t.datetime "created_at",  null: false
-    t.datetime "updated_at",  null: false
+    t.datetime "created_at",              null: false
+    t.datetime "updated_at",              null: false
+    t.string   "custom_manifest_pattern"
```

Comment:
i think `string` defaults to a limit of 255 characters. Is that enough for a IIIF URL?

Reply:
Thanks for catching this. I've updated to use `:text`